src/test/java/de/uhd/ifi/se/decision/management/jira/extraction/gitclient/TestSetUpGit.java

```diff
@@ -70,8 +70,30 @@ public static void setUpBeforeClass() {
 		makeExampleCommit("readMe.txt", "TODO Write ReadMe", "Init Commit");
 		makeExampleCommit();
 		makeExampleCommit("GodClass.java",
-				"public class GodClass {"
-						+ "//@issue Small code issue in GodClass, it does nothing. \t \n \t \n \t \n}",
+				"// @" + "con This is a structure violation, but it should not kill knowledge extraction\n" + 
```

Comment:
We should find a way to separate such test code and commits from the Java code. {issue}How can we access test files in the test/resources folder?{issue}